.gitignore

```diff
@@ -1,3 +1,4 @@
 config.yaml
 node_modules/
 yarn-error.log
+yarn.lock
```

Comment:
I dont think we should gitignore `yarn.lock` the purpose of `yarn.lock` is so we always run the same package versions in production and in dev. That way we wouldnt get issues leftpad. `yarn.lock` has almost no issues unlike `gem.lock` which is all issues. 

Either way, i dont think we should have that there.